scripts/utils/trading_strategy_helpers.js

```diff
@@ -429,30 +429,35 @@ withdrawal of or to withdraw the desired funds
   const buildTransferApproveDepositFromOrders = async function(
     masterAddress,
     bracketAddresses,
+    targetTokenAddress,
     stableTokenAddress,
+    lowestLimit,
+    highestLimit,
+    targetPrice,
     investmentStableToken,
-    targetTokenAddress,
     investmentTargetToken,
     storeDepositsAsFile = false
   ) {
     const fleetSize = bracketAddresses.length
-    assert(fleetSize % 2 == 0, "Fleet size must be a even number")
+    const stepSizeAsMultiplier = Math.pow(highestLimit / lowestLimit, 1 / bracketAddresses.length)
+    const bracketIndexAtTargetPrice = Math.floor(Math.log(targetPrice / lowestLimit) / Math.log(stepSizeAsMultiplier))
+    assert(bracketIndexAtTargetPrice <= fleetSize, "TargetPrice is not between the bounds")
+
```

Comment:
There are two assertions that we could make:
if `bracketIndexAtTargetPrice==0` then `investmentStableToken` must be zero, since no bracket will receive `stableToken`.
if `bracketIndexAtTargetPrice==fleetSize` then `investmentTargetToken` must be zero, since no bracket will receive `targetToken`.